articles/active-directory/saas-apps/lessonly-tutorial.md

```diff
@@ -121,17 +121,17 @@ In this section, you enable Azure AD single sign-on in the Azure portal and conf
 	b. In the **Identifier** textbox, type a URL using the following pattern:
 	| |
 	|--|
-	| `https://<companyname>.lesson.ly/auth/saml/metadata`|
+	| `https://<companyname>.lessonly.com/auth/saml/metadata`|
```

Comment:
Same here @susanesho! Just curious 👍

Reply:
Thanks for noting this, duplicated urls will be removed 👍